### DISCUSSION TITLE
User permissions, launch only playbooks

### DISCUSSION THREAD

**Discussion by len-ro** [semaphoreui/semaphore#1052] [Ideas]
Hello,

I am looking for a solution which would allow the delegation of some tasks (via ansible playbooks) to other member teams (support, operations). I've tested semaphore and it has a nice and clean interface unlike some other tools I tested. It is also simple to install and use. However all users have the same rights. The devops team has no problem running the playbooks from the terminal, the purpose of an interface would be to give access to some users just to execute a deployment, a restart, or other operation. Can this be achieved with semaphore? It seems a launch-only branch was available at some point but it has been scrubbed since.

Thank you.

**Comment by ccrembil** [comment]
I found your feature as a config.json parameter "non_admin_can_create_project". 
Can also be configured as env variable "SEMAPHORE_NON_ADMIN_CAN_CREATE_PROJECT". 
Just turn it to "no" and non-admin users can't create new local projects.
